### PR TITLE
add header and hero section in the home page

### PR DESCRIPTION
**Pull request by AhmedMohsen600**
in this branch we:

       - Added linters for HTML&CSS
       - Added lighthouse library 
       - Created index.html file 
       - Added header & hero section
       - created style.css file
       - styled the header & hero section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-Microverse
+
+> This is just a simple project under the supervision of the Microverse platform that tests how descriptive our repo instruction (pull requests description, readme file), also Linters were installed in the project to check for internal code errors and make the code as clean as possible.
+
+# Project Name
+
+> Portfolio: setup and mobile first
```
Comment (HAMMAS-SALEEM) [inline README.md:9]: - [ ] These things are unnecessary the making your readme looking not clean Kindly remove them :+1:

File: README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+## Acknowledgments
+
+- Microverse documentation.
+
+## :memo: License
+
+You can freely use this project for ethical use and educational purposes.
+GitHubGitHub
+AhmedMohsen600 - Overview
+AhmedMohsen600 has 34 repositories available. Follow their code on GitHub.
```
Comment (HAMMAS-SALEEM) [inline README.md:57]: - [ ] Kindly add a license file in your project using these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :+1:
- [ ] After adding the license file in your project kindly add the link of your license file here :+1:

File: index.html [hunk 3]
```diff
@@ -1,12 +1,54 @@
...
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="skills">
+          <button class="btn-skill">html</button>
+          <button class="btn-skill">css</button>
+          <button class="btn-skill">javaScript</button>
+        </div>
+        <button class="view-project">See project</button>
+      </section>
```
Comment (HAMMAS-SALEEM) [inline index.html:51]: ### Your Section
Screenshot from 2022-09-21 17-49-44



- [ ] You have added a wrong section. The section which you have added is for the next project. But for this project you need to add the section which is shown below :arrow_down:

expected

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Welcome to Microverse :tada: :tada: :bouquet: :bouquet:
# Required Changes :recycle:

almost-there

## Hey @AhmedMohsen600 :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Following Linters
:heavy_check_mark: Following Some Requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (AhmedMohsen600) [reply index.html:51]: @HAMMAS-SALEEM  I have the option to choose my template it's mentioned in the requirements!

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @AhmedMohsen600 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -1,12 +1,71 @@
...
+          me.
+        </p>
+        <a class="connect" href="#">let's connect</a>
+        <div class="social-icons">
+          <img class="icon" src="./imgs/twitter.svg" alt="twitter" />
+          <img class="icon" src="./imgs/linked-in.svg" alt="linked-in" />
+          <img class="icon" src="./imgs/just-m.svg" alt="just-m" />
+          <img class="icon" src="./imgs/github.svg" alt="github" />
+          <img class="icon" src="./imgs/finger.svg" alt="finger" />
+        </div>
```
[the remark this answers is not in the document]
Comment (AhmedMohsen600) [reply index.html:44]: @bobb-Rob  Thank you I'm going to do it

**Pull request merged**